src/reducers/extensions.js

```diff
@@ -1,78 +0,0 @@
-/*
-Copyright 2019-2021 The Tekton Authors
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-import { combineReducers } from 'redux';
-import keyBy from 'lodash.keyby';
-
-import {
-  createErrorMessageReducer,
-  createIsFetchingReducer
-} from './reducerCreators';
-
-const type = 'Extension';
-
-export function mapResourceExtension(extension) {
```

Comment:
@AlanGreene is it safe to remove the `mapResourceExtension` and `byName` functions in this reducer?

Reply:
Once the logic in src/containers/Extensions/Extension.js that was relying on the `extensionType` is updated it should be safe to remove 👍